src/states/wisconsin/main.py

```diff
@@ -91,6 +93,7 @@ def aggregate(pdf_data, qry_data):
     'mailingAddress': 'address',
     'clerkName': 'official',
     'jurisdictionName': 'locale',
+    'hindi': 'fipscode',
```

Comment:
Although we're not necessarily using `fipscode` for Wisconsin, I'm using this terminology here so we have a easier time creating functions that works on both states at the front end.

By the way, I didn't rename `fipscode` in Michigan to something more universal (like `code`) because apparently my IP is on a blocklist. It seems they don't allow foreigners to access their election official data, so every time I ran `inv collect michigan` I never get any result (discovered this while trying to access https://mvic.sos.state.mi.us which should probably work fine on your network)